crates/pep508-rs/src/verbatim_url.rs

```diff
@@ -93,7 +93,10 @@ impl VerbatimUrl {
             url.set_fragment(Some(fragment));
         }
 
-        Self { url, given: None }
+        Self {
+            url,
+            given: Some(path_str.to_string()),
```

Comment:
@charliermarsh Do you know why this used to be `None`?

Reply:
Callers are expected to set this using `with_given`, can we do that instead?